app/assets/v2/js/wallet.js

```diff
@@ -46,8 +46,15 @@ function initWallet() {
       options: {
         id: 'b2345081-a47e-413a-941f-33fd645d39b3'
       }
-    }
+    },
+    walletlink: {
+      package: CoinbaseWalletSDK,
```

Comment:
eslint is complaining about `package` not being quoted (because of this rule: https://eslint.org/docs/rules/quote-props):

```suggestion
      'package': CoinbaseWalletSDK,
```

Reply:
thank you @gdixon, now I know why the others were quoted :)